azure_li_services/units/system_setup.py

```diff
@@ -239,6 +239,12 @@ def set_stonith_service(config):
         Command.run(
             ['rescan-scsi-bus.sh']
         )
+        Command.run(
```

Comment:
why was this not an issue in previous images ?
sles12 sp4 images don't have this issue.

Reply:
I can only guess this. My current guess is that the script behind ```rescan-scsi-bus.sh``` did not work the same way between sle12 and sle15. I know that this script is a mess and nobody touches it because people fear to break something. As I also will not touch the script I'm open to restart the services. This will not cause any harm and is backward compatible